packages/twenty-front/src/modules/object-record/object-filter-dropdown/hooks/__tests__/useFilterDropdown.test.tsx

```diff
@@ -178,35 +178,35 @@ describe('useFilterDropdown', () => {
     });
   });
 
-  it('should set objectFilterDropdownSelectedEntityId', async () => {
-    const mockResult = 'value';
-    const { result } = renderHook(() => {
-      useFilterDropdown({ filterDropdownId });
-      const { objectFilterDropdownSelectedEntityIdState } =
-        useFilterDropdownStates(filterDropdownId);
-
-      const [
-        objectFilterDropdownSelectedEntityId,
-        setObjectFilterDropdownSelectedEntityId,
-      ] = useRecoilState(objectFilterDropdownSelectedEntityIdState);
-      return {
-        objectFilterDropdownSelectedEntityId,
-        setObjectFilterDropdownSelectedEntityId,
-      };
-    }, renderHookConfig);
-
-    expect(result.current.objectFilterDropdownSelectedEntityId).toBeNull();
-
-    act(() => {
-      result.current.setObjectFilterDropdownSelectedEntityId(mockResult);
-    });
-
-    await waitFor(() => {
-      expect(result.current.objectFilterDropdownSelectedEntityId).toBe(
-        mockResult,
-      );
-    });
-  });
+  // it('should set objectFilterDropdownSelectedEntityId', async () => {
```

Comment:
Ok we should get rid of `objectFilterDropdownSelectedEntityId` state, but we should rewrite this test for `objectFilterDropdownSelectedRecordIdsState`

Reply:
So what should I in this case, keep test cases as it is and create a new issue to rewrite the tests?
What do u suggest?